code/src/index.css

```diff
@@ -1,13 +1,449 @@
 body {
```

Comment:
There is indentation and extra lines throughout your css that needs to be fixed. And maybe remove the code that you are not using for example the code referring to checkboxes?
 I really appreciate when you add your comments on your cards - that helps me to understand what your code does! 🙌